### PR TITLE
Added fake ban and unbox messages. Added profile changer. Added new s…

### PR DESCRIPTION
**Pull request by BroOfBros**
…kins and knives (Creds to DrunkenBlyetHvH). Fixed third person toggle.

### CODE REVIEW

File: src/Hacks/thirdperson.cpp [hunk 1]
```diff
@@ -5,6 +5,7 @@
 #include "../interfaces.h"
 
 QAngle fakeang= {0.0f, 0.0f, 0.0f}, realang = {0.0f, 0.0f, 0.0f};
+static bool buttonToggle = false;
```
Comment (acuifex) [inline src/Hacks/thirdperson.cpp]: Global variable already has static duration. no need to explicitly write it here.

**Comment by acuifex** [comment]
other than this, that is something that lwss would add to fuzion. thanks.

**Comment by audadhf** [comment]
everyone can see the fake ban? this wasn't patched?

**Comment by acuifex** [comment]
removed some stuff that i didn't like or looked a bit not optimal.

**Pull request merged**